docs/scripting/scripts.lib/can_continue_execution_error.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: scripts.lib().can_continue_execution_error()
+---
+
+A function that checks if there is enough time remaining for execution based on a specified number of milliseconds. If not enough time remains, the function returns an error.
+
+## Syntax
+
+```js
+#fs.scripts.lib().can_continue_execution_error(remain_threshold);
+```
+
+### Parameters
+
+# remain_threshold
+
+A number in milliseconds.
+
+### Return
+
+If there is not enough time remaining the function returns an object. Otherwise, it will return `null`.
```

Comment:
```suggestion
If there is not enough time remaining the function returns an object:

{ok: false, msg: name + " could not continue executing because " + remain_threshold + "ms are not available for execution."}

Otherwise, it will return `null`.
```